src/styles/main.scss

```diff
@@ -1,3 +1,49 @@
+$col: 100% 1, 80% 2, 60% 3, 40% 4, 20% 5, 0 6;
+$box-width: 300px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_antisnake/blob/master/checklist.md)
add variables.scss, reset.css, `snake.scss`(or smth like that)